src/openms/source/IONMOBILITY/IMDataConverter.cpp

```diff
@@ -124,7 +124,7 @@ namespace OpenMS
     prototype.clear(false);
 
     // adds a new spectrum with drift time to `out`
-    auto addBinnedSpec = [&out, &im_frame, im_unit = im_unit, &prototype](double drift_time_avg) {
```

Comment:
I am not so sure about this one.

Reply:
sure if it is not used it can be removed